### ISSUE TITLE
(medium) [Planning] Checklists section

### ISSUE THREAD

**Issue by yatil** [w3c/wai-media-guide#62]
#### Intro

* The checklists intro is not clear.
* Do they really include all user needs or just the accessibility features we defined. As with all accessibility recommendations there are likely users who have different needs.

I suggest removing the heading and summarizing the lists/sentences:

> Find accessibility features for audio-only and video content below. Note how the requirements change whether the auto/video is a recording (“pre-recorded”) or streamed live. [probably unnecessary:] Follow the links to find more information on how to implement specific accessibility features.

#### Inform users when not needed

* I think the examples are too wordy. Usually you would use captions such as [music playing] or [inaudible] instead of describing why there are no captions. When a video has no sound, one would just remark ”no sound”.

* This section does not describe where to provide this information. If a site usually has videos with captions, it would be good to have that information in the caption directly because users might not be able to find it in time.

#### Provide both captions and a transcript

* I would be conscious to say it is easy to provide a transcript if you have captions and vice-versa. If you have captions, you still need a special field in the CMS and maybe an expand/collapse section on the page. 
* I think this section should not bury the lede and start with the important information: <q>Captions and transcripts use the same text. Therefore, you should provide both.</q>

**Comment by shawna-slh** [comment]
> Find accessibility features for audio-only and video content below. Note how the requirements change whether the auto/video is a recording (“pre-recorded”) or streamed live. 

That seems less clear to me. :-) I added a sentence and changed the bullets. Hopefully clearer to both of us now. :-)

re "[probably unnecessary:] " -- actually it is. That was added in response to a few comments about it not being clear where the links go. :-)

**Comment by shawna-slh** [comment]
> If a site usually has videos with captions, it would be good to have that information in the caption directly

True. And if you're skilled and have the tools, it's fairly easy to add a caption file with that. However, if you're not, then that's probably too much of a burden.

So I'll put both options. :-)

> Users who need captions will look there, so you can provide a captions file with only the appropriate indication, such as "[background music]". Or you can provide the information in text with the video, such as:

Also trimmed down the examples.

**Comment by shawna-slh** [comment]
> [Provide both captions and a transcript](https://wai-media-guide.netlify.com/design-develop/media/planning/#captions-and-transcript)

It starts with the user needs -- to show readers ***why* it is important** -- and hopefully convince them to do it.

It has bolded at the end "**Captions and transcripts use the same text. Once you have one, it’s fairly easy to develop the other.**"
